### PR TITLE
feat: :sparkles: add a setting for rest time

### PR DESCRIPTION
**Pull request by mike011**
## Description
In settings you can now pick a rest time in minutes and seconds. These values are stored to the AppStorage under `restTimeMinutes` and `restTimeSeconds`. SwiftUI currently doesn’t have a time picker that doesn’t show AM / PM. I could’ve used a UIKIT picker, but I thought this looked good enough.

### Related Issue
https://github.com/heyjaywilson/peakfit/issues/22

## How Has This Been Tested?
Picked various values from the settings screen and varied they are still set on relaunch and navigating back and forth. 

## Screenshots (if appropriate):

https://github.com/user-attachments/assets/e1bff3f0-ddd2-4e95-ada1-7663ca13bcbf

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Developer tooling (fix or improve developer tooling)

## Checklist:

- [x] My code follows the code style of this project.
- [] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: AppPackages/Settings/Sources/Settings/SettingsView.swift [hunk 1]
```diff
@@ -14,16 +14,33 @@ import SwiftUI
...
-		NavigationStack {
-			Text("This is the Settings View")
-				.navigationTitle("Settings")
-		}
-	}
+    public init() {}
+
+    public var body: some View {
+        NavigationStack {
+            HStack {
```
Comment (heyjaywilson) [inline AppPackages/Settings/Sources/Settings/SettingsView.swift:24]: Nice work and looks good to me!

Do you mind putting it inside a `List`?

**Review by heyjaywilson — CHANGES_REQUESTED** [review]
@mike011 just asking for a small change hopefully

File: AppPackages/Settings/Sources/Settings/SettingsView.swift (hunk 1 above)
Comment (mike011) [reply AppPackages/Settings/Sources/Settings/SettingsView.swift]: That does look much better.

File: PeakFit.xcodeproj/project.pbxproj
Comment (heyjaywilson) [inline PeakFit.xcodeproj/project.pbxproj:1]: issue: this shouldn't be committed. Can you remove this file?

File: PeakFit.xcodeproj/xcshareddata/xcschemes/PeakFit.xcscheme
Comment (heyjaywilson) [inline PeakFit.xcodeproj/xcshareddata/xcschemes/PeakFit.xcscheme:1]: can you also remove this file from the commit?

**Review by heyjaywilson — CHANGES_REQUESTED** [review]
Nice work. Just some files that shouldn't be committed.

**Comment by mike011** [comment]
> Nice work. Just some files that shouldn't be committed.

Oops, sorry about that, should be fixed now.

**Review by heyjaywilson — APPROVED** [review]
Perfect! Thanks for those changes!

**Pull request merged**